pkgs/os-specific/linux/systemd/default.nix

```diff
@@ -89,6 +88,7 @@
 , withCompression ? true  # adds bzip2, lz4, xz and zstd
 , withCoredump ? true
 , withCryptsetup ? true
+, withRepart ? true
```

Comment:
If you want to minimize it further, you can add a `withSysupdate` flag ;)

Reply:
I guess :>